pkg/awsds/sessions_test.go

```diff
@@ -163,6 +163,31 @@ func TestNewSession_AssumeRole(t *testing.T) {
 		require.NotNil(t, sess)
 		assert.Equal(t, "me-south-1", *sess.Config.Region)
 	})
+
+	t.Run("Assume role is enabled with a gov region", func(t *testing.T) {
+		resetEnvironmentVariables()
+		fakeNewSTSCredentials := newSTSCredentials
+		newSTSCredentials = func(c client.ConfigProvider, roleARN string,
+			options ...func(*stscreds.AssumeRoleProvider)) *credentials.Credentials {
+			sess := c.(*session.Session)
+			// Verify that we are using the well-known region
+			assert.Equal(t, *sess.Config.Region, "us-gov-east-1")
```

Comment:
```suggestion
			assert.Equal(t, "us-gov-east-1", *sess.Config.Region)
```
should be in this order for this library. (seems to be mistakenly in reverse order in other tests too)